ref(projects): extra_fields to partners.project_list (#62)
ulule/developers.ulule.com: content/reference/project.md

## List all partner projects

Retrieves all the projects that have a partnership with the partner with the given slug.

> Projects without status `online` are only accessible to partner users.

Reviewer: ```suggestion
This endpoint is only accessible to partner users, unless the `status=online` query parameter is present.
```

| --------- | ------------ |
| `:slug`   | Partner slug |

> | Parameter   | Description                                                                                                                                        |

Reviewer: we usually add a `Query parameters` subsection, and a short text such as `The list can be filtered with the following query parameters:`

| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `status`    | Filter the project list. Can be one of `online`, `new`, `pending`, `pending-owner`, `validated`, `refused`, `waiting`, `pending-final-validation`. |

Reviewer: sort fields alphabetically

> | `status`    | Filter the project list. Can be one of `online`, `new`, `pending`, `pending-owner`, `validated`, `refused`, `waiting`, `pending-final-validation`. |

Reviewer: ```suggestion
| `status`    | Project status. Can be one of `online`, `new`, `pending`, `pending-owner`, `validated`, `refused`, `waiting`, `pending-final-validation`. |
```

> | `status`    | Filter the project list. Can be one of `online`, `new`, `pending`, `pending-owner`, `validated`, `refused`, `waiting`, `pending-final-validation`. |

Reviewer: we usually don't add trailing dots

> | `is_winner` | Filter the project list. Can be `true` or `false`.                                                                                                 |

Reviewer: ```suggestion
| `is_winner` | Whether the project is a prizewinner. Can be `true` or `false`.                                                                                                 |
```
